schema.xsd

```diff
@@ -166,20 +168,52 @@
         <xs:attribute name="relevance" type="xs:decimal" use="required"/>
     </xs:complexType>
 
+    <!-- Range slider filters. Filters "min"/"max" may not have any values if there is nothing to filter. -->
     <xs:complexType name="range">
         <xs:sequence>
-            <xs:element name="min" type="xs:decimal"/>
-            <xs:element name="max" type="xs:decimal"/>
+            <xs:element name="min">
+                <xs:simpleType>
```

Comment:
Define this union type as a separate named one, as we do it with `range`, because it's used in multiple places.